packages/itmat-ui-react/package.json

```diff
@@ -34,8 +32,7 @@
         "react-apollo": "3.1.5",
         "react-dom": "16.13.1",
         "react-router-dom": "5.2.0",
-        "react-scripts": "3.4.0",
-        "subscriptions-transport-ws": "0.9.16"
```

Comment:
why is this removed?

Reply:
It is not used in significant way, in fact it is triggering constant new network requests rather than rely on a single web-socket, it also breaks the proxy/HMR setup in this current configuration.

Reply:
but it is need for the graphql subscription for the job status. what do you mean "significant"

Reply:
https://github.com/dsi-icl/itmat-broker/blob/c8258fc1155f161963b7e2e837b39c798ca67cbf/packages/itmat-ui-react/src/components/datasetDetail/tabContent/dashboard/jobs.tsx#L34-L51

Reply:
if the package is removed how would this work?